src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonWriterOptionsTests.cs

```diff
@@ -36,26 +36,80 @@ public static void JsonWriterOptionsCtor()
         }
 
         [Theory]
-        [InlineData(true, true, 0)]
-        [InlineData(true, false, 1)]
-        [InlineData(false, true, 1024)]
-        [InlineData(false, false, 1024 * 1024)]
-        public static void JsonWriterOptions(bool indented, bool skipValidation, int maxDepth)
+        [InlineData(true, '\t', 1, true, 0)]
+        [InlineData(true, ' ', 127, false, 1)]
+        [InlineData(false, ' ', 0, true, 1024)]
+        [InlineData(false, ' ', 4, false, 1024 * 1024)]
+        public static void JsonWriterOptions(bool indented, char indentCharacter, int indentSize, bool skipValidation, int maxDepth)
         {
             var options = new JsonWriterOptions();
             options.Indented = indented;
+            options.IndentCharacter = indentCharacter;
+            options.IndentSize = indentSize;
             options.SkipValidation = skipValidation;
             options.MaxDepth = maxDepth;
 
             var expectedOption = new JsonWriterOptions
             {
                 Indented = indented,
+                IndentCharacter = indentCharacter,
+                IndentSize = indentSize,
                 SkipValidation = skipValidation,
                 MaxDepth = maxDepth,
             };
             Assert.Equal(expectedOption, options);
         }
 
+        [Theory]
+        [InlineData(true, '\t', 1, true, 0)]
+        [InlineData(true, ' ', 127, false, 1)]
+        [InlineData(false, ' ', 0, true, 1024)]
+        [InlineData(false, ' ', 4, false, 1024 * 1024)]
+        public static void JsonWriterOptions_Properties(bool indented, char indentCharacter, int indentSize, bool skipValidation, int maxDepth)
+        {
+            var options = new JsonWriterOptions();
+            options.Indented = indented;
+            options.IndentCharacter = indentCharacter;
+            options.IndentSize = indentSize;
+            options.SkipValidation = skipValidation;
+            options.MaxDepth = maxDepth;
+
+            Assert.Equal(indented, options.Indented);
+            Assert.Equal(indentCharacter, options.IndentCharacter);
+            Assert.Equal(indentSize, options.IndentSize);
+            Assert.Equal(skipValidation, options.SkipValidation);
+            Assert.Equal(maxDepth, options.MaxDepth);
+        }
+
+        [Fact]
+        public static void JsonWriterOptions_MultipleValues()
+        {
+            JsonWriterOptions defaultOptions = default;
```

Comment:
Could you also add a test that simply checks that `default(JsonWriterOptions)` returns the expected indent character and size? Thanks

Reply:
Test added.